### PR TITLE
POL-37 Check for Unencrypted S3 buckets

### PR DESCRIPTION
**Pull request by chandravurubindi**
Initial version for POL-37.

### Description

This policy checks all S3 buckets in the AWS account and reports on any that do not have default encryption set. When an unencrypted S3 bucket is detected, the user can choose to enable default encryption (AES-256) and the user can also choose to delete the unencrypted S3 Buckets by enabling 'delete action' as mentioned in Enable delete action section below.

### Issues Resolved
When an unencrypted bucket is detected, an email action is triggered automatically to notify the specified users of the incident. Users then have an option to modify configuration after manual approval, and even Users can perform delete action if required.

- encrypt - modifies the configuration of the unencrypted S3 Bucket by enabling the Default encryption i.e. AES-256

### Contribution Check List

- [x] All tests pass.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### Running Template link:
Delete S3 Bucket: https://governance.rightscale.com/org/1105/projects/60073/policy-incidents/5d354fab89b09a0081d321f5
Update Default Encryption: https://governance.rightscale.com/org/1105/projects/60073/policy-incidents/5d3546b56d4c6800813e1d9a

### Screenshot:
Encrypt_Action_POL-37
Delete_Action_POL-37

### CODE REVIEW

File: security/aws/unencrypted_s3_buckets/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+* encrypt - modifies the configuration of the unencrypted S3 Bucket by enabling the Default encryption i.e. AES-256
+ 
+### Input Parameters
+ 
+- *Email addresses of the recipients you wish to notify* - A list of email addresses to notify
+- *Ignore tags* - S3 Buckets with any of these tags will be ignored 
+ 
+### Required RightScale Roles
+ 
+- policy_manager
```
Comment (rshade) [inline security/aws/unencrypted_s3_buckets/README.md]: We no longer need to have policy_manager here.

Comment (chandravurubindi) [reply security/aws/unencrypted_s3_buckets/README.md]: I had taken care this, thanks.

**Comment by chandravurubindi** [comment]
@cdwilhelm - Can you please review it and approve the same

**Comment by chandravurubindi** [comment]
We are considering @cdwilhelm approval from the phoenix team, hence merging this pr to master branch.

**Pull request merged**